api/app/signals/apps/signals/migrations/0156_alter_attachment_options.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Vereniging van Nederlandse Gemeenten
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('signals', '0155_alter_attachment_options'),
+    ]
+
+    operations = [
```

Comment:
The translations are good 👍🏼 

However, like I said when we talked about changing them to Dutch, this will not create a migration that alters the permissions in the database.

So we still have the English translations in the database.

Before:
![Screenshot from 2022-04-19 09-17-47](https://user-images.githubusercontent.com/23056530/163948569-39d9ba63-796f-4ef7-92e5-a8faca816611.png)

Migration:
![Screenshot from 2022-04-19 09-19-37](https://user-images.githubusercontent.com/23056530/163948612-57b1f54a-fd41-42c5-8040-4daa220fd8d6.png)

After:
![Screenshot from 2022-04-19 09-20-12](https://user-images.githubusercontent.com/23056530/163948630-cd139b08-84d2-4229-af06-bea97ff79e59.png)

See https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/signals/migrations/0104_auto_20200410_1001.py where we also changed the English translations in the database to dutch.

Reply:
@tcoenen because I heard you are sick I fixed the issue in the translations of the newly added permissions

![Screenshot from 2022-04-21 14-18-18](https://user-images.githubusercontent.com/23056530/164456558-ea614462-40c5-4046-b9ff-961985e40cf1.png)

Reply:
@tcoenen because I heard you are sick I fixed the issue in the translations of the newly added permissions

![Screenshot from 2022-04-21 14-18-18](https://user-images.githubusercontent.com/23056530/164456558-ea614462-40c5-4046-b9ff-961985e40cf1.png)